### ISSUE TITLE
U(AT) SSFA Reference Number

### ISSUE THREAD

**Issue by KKaselkar** [unicef/etools-issues#769]
Reference number for SSFA needs to be same as SSFA Header in the agreement.

**Comment by ewheeler** [comment]
new requirement, so labelling as enhancement

**Comment by KKaselkar** [comment]
however the requirements document specify that the and SSFA header (agreement) and SSFA is the same document, and this is one of the main reason for the change in handling SSFA in etools

**Comment by ewheeler** [comment]
@KKaselkar i can't find in requirements or any SSFAs with ref number mismatch. please provide details

**Comment by KKaselkar** [comment]
In PMP Release 3 Remediation Functional Specifications Requirements document in section 2.2 paragraph 1 it is mentioned it is one document.

**Comment by ewheeler** [comment]
yes, one document. but specification lacks any requirements for these ref numbers

**Comment by ShrinivasPuranik** [comment]
Usability issue. Future release. 
Pending discussion.

**Comment by mpawlowski85** [comment]
Need to discuss options to solve the issue

**Comment by mpawlowski85** [comment]
Technical solution is agreed. 
The SSFA Agreement number will be the same as the intervention . Amendment is added at the intervention level and the agreement will inherit the updated number from intervention. 

For Legacy data: 
Draft / Signed / Active / Ended / Suspended SSFA interventions will follow the new reference rules once it is implemented.
